src/dynamoDAO/DynamoDBDAO.go

```diff
@@ -106,3 +104,49 @@ func DeleteAll(clientConfig *dynamodb.Client, tableName string) {
 		}
 	}
 }
+
+// DeleteExpiredUUIDs Expired UUIDs have persisted for 30 days or longer
+func DeleteExpiredUUIDs(clientConfig *dynamodb.Client, tableName string) {
+
+	loc, _ := time.LoadLocation("UTC")
+
+	earliestAcceptedDate := time.Now().In(loc)
+	latestAcceptedDate := time.Now().In(loc).Add(-720 * time.Hour) // 30 days from current time
+
+	scan := dynamodb.NewScanPaginator(clientConfig, &dynamodb.ScanInput{
```

Comment:
Scans are extremely costly especially at scale. 
These should be avoid at all costs and do not have to be used here

Reply:
You should have a gsi that you can query on to partition uuids by the date so you can get that result back the delete each one.